source/develop/release-management/features/gluster/gluster-reset-brick-support.html.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Gluster Reset Brick Support
+category: feature
+authors: godas
+feature_name: Gluster management
+feature_modules: engine,gluster
+feature_status: Completed
+---
+
+# Gluster Support
+
+## Summary
+
+This feature provides support for reset-brick Gluster based storage clusters in oVirt. Glusterfs repo is available at <http://download.gluster.org/pub/gluster/glusterfs/>
+
+## Owner
+
+*   Feature owner: Gobinda Das <godas@redhat.com>
+    -   GUI Component owner: Gobinda Das <godas@redhat.com>
+    -   Engine Component owner: Gobinda Das <godas@redhat.com>
+    -   VDSM Component owner: Gobinda Das <godas@redhat.com>
+    -   QA Owner: SATHEESARAN <sasundar@redhat.com>
+
+## Current Status
+
+*   Requires : glusterfs >= 3.9
+
+## Detailed Description
+
+This feature will introduce the capability of reset existing brick from the oVirt UI. Administrator will be able to perform Gluster volume reset-brick operation.
+
+### Approach
+
+With this feature, oVirt will start supporting reset-brick functionalty in case Server has gone bad and got reprovisioned, but it may got its brick and want to use same brick.
+
+
+
+### User Interface
+
+New tabs will be displayed as sub-tabs when user selects a brick from the "Gluster Volume -> Bricks" sub-tab.
+
+
+*   Actions on "Bricks" tab / sub-tab:
+    -   Reset Brick
+
+Once User will click on "Reset Brick", One Confirmation window will pop up.
```

Comment:
Is there any input needed for reset brick? In case if the host's IP address changes, does reset brick take the new brick name, that is, <new server ip address>:/brick-directory as parameter?